Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPIncidents/README.md

```diff
@@ -0,0 +1,71 @@
+Note: Support for this pack moved to the partner on March, 22, 2022.
```

Comment:
The integration doc should be auto-generated. Please remove this file and run the command below to create the doc:

```python
demisto-sdk generate-docs -i Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPIncidents/README.md
```

Reply:
will do that